services/topics/src/index.ts

```diff
@@ -14,35 +14,31 @@
  limitations under the License.
  */
 
- import express, { Application, Request, Response } from "express"
-
- const {  PORT, TOPICS_SERVER_HOST } = process.env
- 
- const app: Application = express()
- 
- app.use((req, res, next) => {
-   next()
- })
- app.use(express.static("src/public"))
- app.set("view engine", "ejs")
- app.set("views", "src/views")
- 
- 
- app.get("/", async (req: Request, res: Response) => {
- 
- 
-   var hostname = req.hostname
-   var title = (hostname).substring(0, hostname.indexOf('.'))
- 
-   console.log("TITLE:"+title)
-   const params = {
-     title,
-     TOPICS_SERVER_HOST
-   }
-   res.render("index", params)
- })
- 
- app.listen(PORT, async () => {
-   console.log(`Listening on port ${PORT}`)
- })
- 
\ No newline at end of file
+import express, { Application, Request, Response } from "express"
+
+const { PORT, TOPICS_SERVER_HOST } = process.env
+
+const app: Application = express()
+
+app.use((req, res, next) => {
+  next()
+})
+app.use(express.static("src/public"))
+app.set("view engine", "ejs")
+app.set("views", "src/views")
+
+app.get("/", async (req: Request, res: Response) => {
+  var hostname = req.hostname
+  var title = hostname.substring(0, hostname.indexOf("."))
```

Comment:
in this case, title is always "topics" isn't it ?

Reply:
In prod we will use sub-domains. E.g: soccer.privacy-sandbox-demos-news.dev, so this will be different depending on which page you are on